components/core/src/clp_s/CommandLineArguments.cpp

```diff
@@ -305,8 +305,8 @@ CommandLineArguments::parse_arguments(int argc, char const** argv) {
                     "ordered-chunk-size",
                     po::value<size_t>(&m_ordered_chunk_size)
                             ->default_value(m_ordered_chunk_size),
-                    "Number of records to include in each output file when decompressing records "
-                    "in log order"
+                    "Chunk size (B) for each output file when decompressing records in log order. "
+                    "No chunking is performed when this argument is set to zero."
```

Comment:
```suggestion
                    "Chunk size (B) for each output file when decompressing records in log order."
                    " When set to 0, no chunking is performed."
```

Two changes:
* Move the trailing space to the next line to follow our convention of putting the line continuation (e.g., operator) on the next line.
* Rephrasing (feel free to disagree).